test/project_types/script/forms/create_test.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "project_types/script/test_helper"
+require 'byebug'
```

Comment:
🔥 😄

Reply:
You also shouldn't need to do this since `project_types/script/test_helper` imports `test_helper` which already requires byebug